cypress.json

```diff
@@ -0,0 +1,14 @@
+{
+  "projectId": "rnnyfb",
+  "integrationFolder": "spec/integration",
+  "testFiles": "**/*.feature",
+  "pluginsFile": "spec/integration/cypress/plugins",
+  "supportFile": "spec/integration/cypress/support",
+  "screenshotsFolder": "spec/integration/cypress/screenshots",
+  "videosFolder": "spec/integration/cypress/videos",
+  "videoUploadOnPasses": false,
```

Comment:
@HHogg idea behind it: we only need videos for investigations, so I didn't wanna overload our account. WDYT? should we instead upload every video?